modules/imgproc/src/templmatch.cpp

```diff
@@ -950,12 +950,12 @@ static void common_matchTemplate( Mat& img, Mat& templ, Mat& result, int method,
                 else
                     t = std::sqrt(diff2)*templNorm;
 
-                if( fabs(num) < t )
+                if( fabs(num) < t || method == CV_TM_SQDIFF_NORMED)
```

Comment:
`isNormed` value (line 945) is True for `CV_TM_SQDIFF_NORMED` mode.
Probably this is wrong if `CV_TM_SQDIFF_NORMED` is not a real "normed" mode.